state/model.go

```diff
@@ -871,6 +871,39 @@ func (m *Model) AllUnits() ([]*Unit, error) {
 	return units, nil
 }
 
+type ApplicationEndpointBindings struct {
+	AppName  string
+	Bindings map[string]string
+}
+
+// AllBindings return all endpoint - space bindings deployed.
```

Comment:
// AllEndpointBindings returns all endpoint -> space bindings for every application.

Plus best to use AllEndpointBindings in case there are different bindings